src/etc/inc/openvpn.inc

```diff
@@ -1188,9 +1198,33 @@ function openvpn_reconfigure($mode, $settings) {
 				} else {
 					$tls_directive = "tls-auth";
 					if ($mode == "server") {
-						$tlsopt = 0;
+						switch($settings['tlsauth_keydir']){
+							case '1':
+								$tlsopt = 1;
+								break;
+							case '2':
+								$tlsopt = '';
+								break;
+							case '':
+							case '0':
+							default:
```

Comment:
Since you are using `default:` here you don't need to explicit add `case '':` and `case '0':`

Reply:
in https://github.com/pfsense/pfsense/pull/3999/commits/f93ec3853fc0c01760606994422e9e8fc0d645c9